lib/autoPagination.js

```diff
@@ -0,0 +1,190 @@
+'use strict';
+
+var makeRequest = require('./makeRequest');
+var utils = require('./utils');
+
+function getItemCallback(args) {
```

Comment:
Do you think that we could make an effort to organize this file a little bit more? Either:

1. Functions listed alphabetically.
2. Functions listed by category with exported on the top (so you drop into the file and see the most important ones first) and unexported below, and alphabetically within each category.

Reply:
sure, I like (2)!

It's a bit of a smell in JS to call functions before they're defined, imo (it's not possible with arrow functions like `const fn = () =>`, only function declarations like we have here) but given that we're likely to stay with ES5 source code for some time, I'm happy to make a silver lining out of it.

Reply:
✅